aptible/organizations.go

```diff
@@ -0,0 +1,71 @@
+package aptible
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+)
+
+type Organization struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+type HALOrganizationParent struct {
+	Organizations []Organization `json:"organizations"`
+}
+
+type HALOrganizationResponse struct {
+	Embedded HALOrganizationParent `json:"_embedded"`
+}
+
+// getOrganizations - get all organizations a user has access to
+func (c *Client) getOrganizations() ([]Organization, error) {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/organizations", GetAuthURL()), nil)
+	//Handle Error
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.RawToken))
+	req.Header.Set("Accept", "application/json")
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, err
```

Comment:
may want to case switch instead of blindly bubbling, may loop back for slightly better errors, but the simplest error returns seem to work okay and be readable:

<img width="723" alt="image" src="https://user-images.githubusercontent.com/2961973/198020600-c895ba7b-f445-466d-88d6-13b77effa4e9.png">


and/or 

<img width="1334" alt="image" src="https://user-images.githubusercontent.com/2961973/198021259-0e8c384a-03c0-4b93-a1e6-f968627892ae.png">